corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -174,13 +176,19 @@ def get_sql_chunks(run_config):
 
     @staticmethod
     def _yield_missing_in_es(chunk):
```

Comment:
Is the idea that with this change, a `DataRow` will be emitted for each case that is missing in either the hqcase or case_search index? And if it's missing in both it's only emitted once since a `DataRow` doesn't include info about which index it's missing in?

Reply:
yes, and because the republishing to kafka is not index specific either